### PR TITLE
fix: questioning a few lines that force an ORDER clause

### PR DESCRIPTION
**Pull request by mistercrunch**
### SUMMARY

I'm questioning the need for these 2 lines here, and would like to understand what they are about. It seems like this should be handled by the caller if/when needed.

There's nothing really wrong about doing the unnecessary sort apart for the extra cost on the database backend. In the case of the line chart, it may make more sense to sort on time DESC [if anything] to make the forced row LIMIT to show something more consistent.

## before
Screen Shot 2020-10-03 at 9 39 34 AM

## after
Screen Shot 2020-10-03 at 9 33 47 AM

### CODE REVIEW

**Review by villebro — APPROVED** [review]
I agree with removing this, as the current logic causes a funny from-top-to-bottom sweeping effect (I noticed this once when playing around with low limits). A few alternatives could be ordering by groupby, then `__timestamp` in descending order so as to get 1) as many complete series and 2) have them start from most recent datapoint like you mentioned. However, I feel we can get back to this later once when we start implementing server side pagination to chart data requests.

**Comment by mistercrunch** [comment]
Can you think of other things that may break from this? Why is it there in the first place? Maybe the table viz?

**Comment by mistercrunch** [comment]
@villebro ^^^ ?

**Comment by mistercrunch** [comment]
I checked and this changes the default behavior of the "Table" visualization where currently, by default is sorts descending on the first metric.

I feel like the right thing to do is to identify the visualization that do rely on this behavior and to explicitly do it for those.

**Comment by mistercrunch** [comment]
I added the default behavior logic of sorting by main metric DESC in Table when not specified as part of that visualization logic.

**Comment by villebro** [comment]
@mistercrunch sorry, I've been having trouble with notifications on GH. I'll review this asap.

File: superset-frontend/src/views/CRUD/data/database/DatabaseList.tsx [hunk 1]
```diff
@@ -180,7 +180,7 @@ function DatabaseList({ addDangerToast, addSuccessToast }: DatabaseListProps) {
         Header: (
           <TooltipWrapper
             label="allow-dml-header"
-            tooltip={t('Allow Data Danipulation Language')}
+            tooltip={t('Allow Data Manipulation Language')}
```
Comment (mistercrunch) [inline superset-frontend/src/views/CRUD/data/database/DatabaseList.tsx:183]: Bycatch!

**Review by villebro — APPROVED** [review]
LGTM, and I agree that going forward individual viz' should do this type of custom ordering if needed in `buildQuery`.

**Pull request merged**